source/mobile/mobile-faq.rst

```diff
@@ -113,6 +112,74 @@ The following options are available for securing your push notification service:
 
 .. Note:: For configuration details, see guides for :doc:`deploying the Mattermost App Store and Google Play apps <mobile-appstore-install>` and :doc:`deploying your own version of the apps <mobile-compile-yourself>`.
 
+How do I deploy Mattermost with Enterprise Mobility Management providers?
+--------------------------------------------------------------------------
+
+Mattermost enables customers with high privacy and custom security requirements to deploy mobile applications and push notification services using keys that they alone control.
+
+:doc:`Learn more about using AppConfig for EMM providers <mobile-appconfig>`.
+
+How do I host the Mattermost push notification service?
+----------------------------------------------------------
+
+First, you can use the :doc:`Mattermost Hosted Push Notification Service (HPNS) <mobile-hpns>`. Organizations can also `host their own push proxy server instead <https://developers.mattermost.com/contribute/mobile/push-notifications/service/>`_. This is applicable when you want to:
+
+1. customize the `Mattermost mobile apps <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_,
+2. deploy your own push notification service, or
+3. repackage the mobile apps with BlueCedar or AppDome, which some organizations have successfully deployed with but is not officially supported.
+
+How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server?
+--------------------------------------------------------------------------------------------------------------------
+
+When your IT policy requires a corporate proxy to scan and audit all outbound traffic the following options are available:
+
+1 - Deploy Mattermost in a proxy-aware configuration with a pre-proxy relay
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Mattermost push notification service is designed to send traffic directly to the `Apple Push Notification Service (APNS) <https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1>`_ and `Google Fire Cloud Messaging (FCM) <https://firebase.google.com/docs/cloud-messaging>`_ services. 
+
+In a proxy-aware configuration, a :doc:`pre-proxy relay <pre-post-proxy>` accepts messages from the `Mattermost Push Proxy <https://developers.mattermost.com/contribute/mobile/push-notifications/service/>`_ and forwards them to a corporate proxy enforcing your internal IT requirements, before transmitting to their final destination.
```

Comment:
@amyblais `:doc:`pre-proxy relay <pre-post-proxy>`` does not render as a link in the docs, would you be able to help investigate why?

Reply:
Trying this https://github.com/mattermost/docs/pull/3217
Not 100% sure